src/screens/UserPortal/People/People.tsx

```
@@ -229,7 +137,7 @@ export default function People(): React.JSX.Element {
           />
         </div>
 
-        <div className={styles.people_content}>
+        <div className={styles.people_content} aria-label={t('peopleTable')}>
           <div className={styles.people_card_header}>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)